src/app/Instance/ErrorHandling/ErrorHandlingEdit.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  ActionGroup,
+  Alert,
+  Button,
+  Form,
+  FormAlert,
+  PageSection,
+} from "@patternfly/react-core";
+import { useTranslation } from "@rhoas/app-services-ui-components";
+import ErrorHandlingCreate from "@app/Instance/ErrorHandling/ErrorHandlingCreate";
+
+export interface ErrorHandlingEditProps {
+  apiError?: string;
+  getSchema: (method: string) => Promise<object>;
+  isLoading: boolean;
+  method?: string;
+  onCancelEditing: () => void;
+  onSubmit: (method?: string, parameters?: object, schema?: object) => void;
+  parameters?: Record<string, unknown>;
+  registerValidateParameters: (validationFunction: () => boolean) => void;
+  schema?: object;
+}
+
+export const ErrorHandlingEdit = ({
+  apiError,
+  getSchema,
+  isLoading,
+  method,
+  onCancelEditing,
+  onSubmit: onErrorHandlingSubmit,
+  parameters,
+  registerValidateParameters,
+}: ErrorHandlingEditProps): JSX.Element => {
+  const { t } = useTranslation(["openbridgeTempDictionary"]);
+
+  const [isSubmitted, setIsSubmitted] = useState<boolean>(false);
+  const [errorHandlingMethod, setErrorHandlingMethod] = useState<
+    string | undefined
+  >(method);
+  const [errorHandlingParameters, setErrorHandlingParameters] =
+    useState(parameters);
+
+  const onErrorHandlingParametersChange = useCallback(
+    (method?: string, parameters?: Record<string, unknown>): void => {
+      setErrorHandlingMethod(method);
+      setErrorHandlingParameters(parameters);
+    },
+    []
+  );
+
+  const onSubmit = useCallback(
+    (event: FormEvent<HTMLFormElement>) => {
+      event.preventDefault();
+      onErrorHandlingSubmit(errorHandlingMethod, errorHandlingParameters);
+      setIsSubmitted(true);
+    },
+    [errorHandlingMethod, errorHandlingParameters, onErrorHandlingSubmit]
+  );
+
+  useEffect(() => {
+    if (isSubmitted) {
+      document
+        .querySelector(".pf-m-error")
+        ?.parentElement?.previousElementSibling?.scrollIntoView({
+          behavior: "smooth",
+          block: "start",
+          inline: "nearest",
+        });
+      setIsSubmitted(false);
+    }
+  }, [isSubmitted]);
+
+  return (
+    <Form
+      className="error-handling-edit-form"
+      autoComplete="off"
+      onSubmit={onSubmit}
+    >
+      {apiError && (
+        <FormAlert>
+          <Alert
+            className="error-handling-edit__alert"
+            ouiaId="error-schema"
+            variant="danger"
+            title={apiError}
+            aria-live="polite"
+            isInline
+          />
+        </FormAlert>
+      )}
+      <ErrorHandlingCreate
+        schemaId={errorHandlingMethod}
+        getSchema={getSchema}
+        registerValidation={registerValidateParameters}
+        onChange={onErrorHandlingParametersChange}
+        parameters={errorHandlingParameters}
+      />
+      <PageSection
+        stickyOnBreakpoint={{ default: "bottom" }}
+        padding={{ default: "noPadding" }}
+        style={{ boxShadow: "none" }}
+      >
```

Comment:
It's sticky when we have Webhook/Kafka topic as selected method, otherwise (the Ignore case) the buttons are shown in this way:
![image](https://user-images.githubusercontent.com/22591802/194293619-73c5f871-82a8-416d-b192-68e6b186375d.png)
This was the best compromise I was able to achieve. Do you think it is ok or do you have any suggestion for structuring differently the page?

Reply:
I had a look and I couldn't find a quick way to accomplish the sticky actions without refactoring the whole instance page layout. Right now I'm not sure if it can be done without custom styling so It could take a while.

I think we can move on and create a separate ticket to address how to improve this.

Reply:
Thanks for your time.
I hope it will be feasible with Patternfly, instead of custom styling.
However I created https://issues.redhat.com/browse/MGDOBR-1129, hope the description is clear enough.